### PR TITLE
Updating drop_old_assembly_databases to deal with assembly renaming. …

### PR DESCRIPTION
**Pull request by thomasmaurel**
…At the moment this subroutine will drop the new handed over database since it's the same as the old one

### CODE REVIEW

File: ensembl_prodinf/handover_tasks.py [hunk 1]
```diff
@@ -320,8 +320,14 @@ def submit_event(spec,result):
...
         get_logger().debug("Submitted event to event handler endpoint")
 
-def drop_old_assembly_databases(old_assembly_db_list,staging_uri):
+def drop_old_assembly_databases(old_assembly_db_list,staging_uri,tgt_uri):
     """Drop databases on a previous assembly from the staging MySQL server"""
+    tgt_url=make_url(tgt_uri)
+    #Check if the new database has the same name as the one on staging. In this case DO NOT drop it
+    #This can happen if the assembly get renamed
+    if tgt_url.database in old_assembly_db_list:
+        get_logger().info("The assembly has been updated but the new database " + str(tgt_url.database) +" is the same as old one ")
```
Comment (marcoooo) [inline ensembl_prodinf/handover_tasks.py]: I would log a Warning instead.

File: ensembl_prodinf/handover_tasks.py [hunk 2]
```diff
@@ -320,8 +320,14 @@ def submit_event(spec,result):
...
+        return
```
Comment (marcoooo) [inline ensembl_prodinf/handover_tasks.py]: In my opinion, you return either 'something' or do not return at all.

File: ensembl_prodinf/handover_tasks.py [hunk 3]
```diff
@@ -320,8 +322,15 @@ def submit_event(spec,result):
...
         get_logger().debug("Submitted event to event handler endpoint")
 
-def drop_old_assembly_databases(old_assembly_db_list,staging_uri):
+def drop_old_assembly_databases(old_assembly_db_list,staging_uri,tgt_uri):
     """Drop databases on a previous assembly from the staging MySQL server"""
+    tgt_url=make_url(tgt_uri)
+    #Check if the new database has the same name as the one on staging. In this case DO NOT drop it
+    #This can happen if the assembly get renamed
+    if tgt_url.database in old_assembly_db_list:
+        get_logger().info("The assembly has been updated but the new database " + str(tgt_url.database) +" is the same as old one")
```
Comment (marcoooo) [inline ensembl_prodinf/handover_tasks.py]:
```suggestion
        get_logger().warn("The assembly has been updated but the new database " + str(tgt_url.database) +" is the same as old one")
```

**Review by marcoooo — CHANGES_REQUESTED** [review]

File: ensembl_prodinf/handover_tasks.py (hunk 2 above)
Comment (thomasmaurel) [reply ensembl_prodinf/handover_tasks.py]: done

File: ensembl_prodinf/handover_tasks.py (hunk 3 above)
Comment (thomasmaurel) [reply ensembl_prodinf/handover_tasks.py]: done but using debug. warn is not in allowed logger level for ES at the moment.

**Review by marcoooo — APPROVED** [review]

**Review by marcoooo — APPROVED** [review]

**Pull request merged**